common/secrets.js

```diff
@@ -129,6 +141,14 @@ const DOTDIGITAL_API = {
         process.env.DOTDIGITAL_PASS || getParameter('dotdigital.api.password'),
 };
 
+/**
+ * Sandbox domain
+ * Used to specify the requested salesforce endpoint
+ * */
+
+const SANDBOX_DOMAIN =
```

Comment:
Might be worth calling this something like `SALESFORCE_SANDBOX_DOMAIN` for clarity as we already have `CONTENT_API_SANDBOX_URL` so there's a bit of ambiguity about which sandbox is which etc.